src/schema/posts.ts

```diff
@@ -304,6 +304,16 @@ export const typeDefs = /* GraphQL */ `
       id: ID
     ): Post!
 
+    """
+    Get post by canonical URL
+    """
+    postCanonical(
```

Comment:
I would suggest this name which is a bit more explicit in my opinion.
I also think we should compare both canonical and the original URL to the given parameter.
What do you think?

```suggestion
    postByUrl(
```

Reply:
Good point, my idea was that the canonical would be leading anyway, but happy to include a URL or Canonical as the query.